cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaListenersReconciler.java

```diff
@@ -442,6 +443,63 @@ protected Future<Void> internalServicesReady()   {
         return Future.succeededFuture();
     }
 
+    /**
+     * Collect all addresses of services related to clusterIP addresses for Statuses, certificates and advertised
+     * addresses. This method for all ClusterIP type listeners:
+     *      1) Collects the relevant addresses of bootstrap service and stores them for use in certificates
+     *      2) Collects the clusterIP addresses for certificates and advertised hostnames
+     *
+     * @return  Future which completes clusterIP service addresses are collected
+     */
+    protected Future<Void> clusterIPServicesReady()   {
+        for (GenericKafkaListener listener : ListenersUtils.clusterIPListeners(kafka.getListeners())) {
+            boolean useServiceDnsDomain = (listener.getConfiguration() != null && listener.getConfiguration().getUseServiceDnsDomain() != null)
+                    ? listener.getConfiguration().getUseServiceDnsDomain() : false;
+
+            String bootstrapAddress = listener.getConfiguration().getBootstrap().getHost();
```

Comment:
I do not think you should take the host. `host` fields should be probably disabled for the ClusterIP listeenrs, or?

Reply:
right! I confuse it with bootstrap url used by client to reach bootstrap service. We should ignore host field